frontend/src/component/playground/Playground/PlaygroundForm/PlaygroundCodeFieldset/PlaygroundCodeFieldset.tsx

```diff
@@ -188,6 +189,8 @@ export const PlaygroundCodeFieldset: VFC<IPlaygroundCodeFieldsetProps> = ({
                 <Autocomplete
                     disablePortal
                     id="context-legal-values"
+                    freeSolo
```

Comment:
can you add more context why those 2 new fields solve the problem? I find it difficult to tell w/o being involved in a debugging session

Reply:
`freesolo` removes the arrow (dropdown) and `filterSelectedOptions` as the name implies. Looks better